PLP/intro_2_3_palindrom.py

```diff
@@ -0,0 +1,21 @@
+
+def check_palindrome(num):
```

Comment:
the algorithm idea is not bad, but adds some extra complexity. Try to think also to another easier way, maybe by exploiting some of the areas which you already used... e.g: num%10 to determine each digit.